scipy/constants/codata.py

```diff
@@ -1181,9 +1550,12 @@ def parse_constants(d):
 for k in _physical_constants_2002:
     if 'magn.' in k:
         _aliases[k] = k.replace('magn.', 'mag.')
-for k in _physical_constants_2006:
```

Comment:
I think you should keep this, and add similar for 2018. Then fix the test so it works with the result. The idea is to keep backwards compatibility of the names so that code doesn't break after a new codata release.